Tools/DataStorage/DataStorage.cs

```diff
@@ -103,6 +229,174 @@ public Class GetClass(string classId)
             return res;
         }
 
+        public List<string> GetYears()
+        {
+            string sql = "SELECT st_year FROM Classes";
+            List<string> list = new List<string>();
+            SqlConnection myConn = new SqlConnection(StationManager.ConnectionString);
+            try
+            {
+                myConn.Open();
+
+                using (SqlCommand command = new SqlCommand(sql, myConn))
+                {
+                    var reader = command.ExecuteReader();
+                    while (reader.Read())
+                    {
+                        string cur = reader.GetString(0).ToString();
+                        list.Add(cur);
+                    }
+
+                    reader.Close();
+                }
+
+                return list;
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show("There's problem with you connection!\n" + ex.Message);
+            }
+            finally
+            {
+                myConn?.Close();
+            }
+
+            return list;
+
+        }
+
+        public List<Teacher> GetTeachers(string year)
```

Comment:
Probably it's better to name it differently, it returns classes, doesn't it?

Reply:
nah, it returns list with teachers which have class in the certain year, but I think I will change this method when I will do connection between teachers and class, don't want to delete because it can be useful in future